docs/pages/installation/eks.md

```diff
@@ -9,106 +9,77 @@ list: include
-    </div></div>
-    </pre>
-
-    _Note: Here the secret name is meshery-token-5z9xj_
-1. Get secret/token:
-
-    <pre class="codeblock-pre">
-    <div class="codeblock"><div class="clipboardjs">
-    $ kubectl describe secret  sa-1-token-5z9xj
-    Name:         meshery-token-5z9xj
-    Namespace:    default
-    Labels:       <none>
-    Annotations:  kubernetes.io/service-account.name: meshery
-                  kubernetes.io/service-account.uid: 397XXX-XXX-XXXX-XXXXX-XXXXX
-
-    Type:  kubernetes.io/service-account-token
-
-    Data
-    ====
-    ca.crt:     1025 bytes
-    namespace:  7 bytes
-    token:      XXXhbGciOiJSUXXXX     </div></div></pre>
-
-1. Generate new kubeconfig yaml file to use as input to Meshery.
-1. Set config Credential using above generate `token`.
-    
-    <pre class="codeblock-pre">
-    <div class="codeblock"><div class="clipboardjs">
-    $ kubectl config set-credentials meshery --token=XXXXX
-
-    o/p:User "meshery" set.
-    </div></div>
-    </pre>
-
-1. Set current context to our new service account `meshery`
-    
-    <pre class="codeblock-pre">
-    <div class="codeblock"><div class="clipboardjs">
-    $ kubectl config set-context --current --user=meshery
-
-    o/p:
-    Context "aws" modified.
-    </div></div>
-    </pre>
-
-1. Generate kubeconfig yaml file to use as input to Meshery.
-    
-    <pre class="codeblock-pre">
-    <div class="codeblock"><div class="clipboardjs">
-    $ kubectl config view --minify --flatten >  config_aws_eks.yaml
-    </div></div>
-    </pre>
-
-Meshery should now be connected with your managed Kubernetes instance. Take a look at the [Meshery guides]({{ site.baseurl }}/guides) for advanced usage tips.-->
+{% include installation_prerequisites.html %}
 
-*Coming soon*
\ No newline at end of file
+## To set up and run Meshery on EKS:
+
+- Connect Meshery to your EKS cluster
+  - [Meshery CLI (mesheryctl)](#connect-meshery-to-azure-kubernetes-cluster)
+  - [eks CLI (eksctl)](https://eksctl.io/introduction/#installation)
+  - [AWS CLI (aws)](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
+- [Install Meshery on your EKS cluster](#install-meshery-into-your-eks-cluster)
+- [Access Meshery's UI](#port-forward-to-the-meshery-ui)
+
+### Connect Meshery to Elastic Kubernetes Cluster
+
+The following set of instructions expects you to have created a EKS cluster in your resource group Configure Meshery to connect to your EKS cluster by executing:
+
+ <pre class="codeblock-pre"><div class="codeblock">
+ <div class="clipboardjs">
+ mesheryctl system config eks
+ </div></div>
+ </pre>
+
+#### Manual Steps
+
+Alternatively, you may execute the following steps to manually configure Meshery to connect to your AKS cluster.
+
+- Install [AWS CLI(aws)](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html), and login
+  to your AWS account. Install [EKS CLI (eksctl)](https://eksctl.io/introduction/#installation) too to access eks cluster easily.
```

Comment:
I might be missing it - where is `eksctl` used in these instructions? If it isn't, please remove this reference.

Reply:
Eksctl isn't used but just came across... Anyway I'll remove it